src/test/scala/org/kibanaLoadTest/helpers/ESArchiver.scala

```diff
@@ -73,12 +73,12 @@ class ESArchiver(config: KibanaConfiguration, bulkSize: Int = 1000) {
     } catch {
       case e: RuntimeException =>
         e.getMessage match {
-          case "createIndex" =>
+          case s"Failed to create index$x" =>
```

Comment:
I added a comment about a missing parameter in the format string. It might be that you need to update this case clause as well.